config/config.yaml

```diff
@@ -89,9 +95,35 @@ MODEL_FOR_RESEARCHER_REPORT: gpt-3.5-turbo-16k
 
 ### choose the engine for mermaid conversion, 
 # default is nodejs, you can change it to playwright,pyppeteer or ink
-# MERMAID_ENGINE: nodejs
+#MERMAID_ENGINE: nodejs
+
 
 ### browser path for pyppeteer engine, support Chrome, Chromium,MS Edge
 #PYPPETEER_EXECUTABLE_PATH: "/usr/bin/google-chrome-stable"
 
-PROMPT_FORMAT: json #json or markdown
\ No newline at end of file
+PROMPT_FORMAT: json #json or markdown
+
+
+####Customizing the LLM used by Action and Role
+MULTI_LLM: True
```

Comment:
Don't set default value in config.yaml which will effect the usage. Suggest split the model-list and role/action's choose in a single file and load in Config. If no custom model setted, use default openai.